oci/spec_opts.go

```diff
@@ -361,6 +381,11 @@ func WithImageConfigArgs(image Image, args []string) SpecOpts {
 				return err
 			}
 			config = ociimage.Config
+
+			// Deserialize the extended image format, but ignore the error
+			// because the default would be `false` anyways and this is not a
+			// required param for Windows or Linux.
+			_ = json.Unmarshal(p, &ociImageExtended)
```

Comment:
Then only run this conditionally, so that we don't do a `json.Unmarshal` if we're not gonna need it;

```suggestion
			if s.Windows != nil {
				// Deserialize the extended image format, but ignore the error
				// because the default would be `false` anyways and this is not a
				// required param for Windows or Linux.
				var im imageExtended
				_ = json.Unmarshal(p, &im)
				argsEscaped = im.Config.ArgsEscaped
			}
```